brainscore_vision/models/resnet50_finetune_cutmix_e3_robust_linf8255_e0_247x234/requirements.txt

```diff
@@ -0,0 +1,7 @@
+torchvision
+torch
+timm
+PIL
```

Comment:
@pellegreene PIL is no longer supported for Python 3 and instead has been replaced by [Pillow](https://pillow.readthedocs.io/en/stable/), same as #946